DEVNOTES.md

```diff
@@ -14,26 +14,28 @@
 ***
 
 
-## Building Ignite
-Ignite follows standard guidelines for multi-module Gradle projects, so it can be easily built using the following command from the project root directory (you can disable the tests when building using `-x test`):
+## Quick Start
+Ignite follows standard guidelines for multi-module Gradle projects, so it can be easily built using the following command from the project
+root directory:
 ```
 gradlew clean build
 ```
-Upon build completion, CLI tool can be found be under `modules/cli/build` directory. Use `ignite` on Linux and MacOS, or `ignite.exe` on Windows.
-***
+Apart from building the project itself this command performs a few additional actions, for instance, testing. The build runs faster if
```

Comment:
This command builds a project and performs a few additional actions, for example it also runs tests. The build runs faster if